Resources/Private/JavaScript/media-module/src/components/SideBarLeft/Tree/AssetCollectionTree.tsx

```diff
@@ -167,7 +167,8 @@ const AssetCollectionTree = () => {
                         assetCollection={assetCollection}
                         onClick={selectAssetCollection}
                         level={1}
-                        isActive={assetCollection.title == selectedAssetCollection?.title && !selectedTag}
+                        isActive={assetCollection.title == selectedAssetCollection?.title}
+                        isFocused={assetCollection.title == selectedAssetCollection?.title && !selectedTag}
```

Comment:
I'd like to suggest adding the same logic to the "All"-Collection as well. Replacing L146 with these two lines should do it:
```typescript
isActive={!selectedAssetCollection}
isFocused={!selectedAssetCollection && !selectedTag}
```

Reply:
Thought about it too, agree :)